src/openforms/formio/api/serializers.py

```diff
@@ -51,3 +51,19 @@ def get_url(self, instance) -> str:
             kwargs={"uuid": instance.uuid},
             request=request,
         )
+
+
+class MapSearchLatLng(serializers.Serializer):
+    lat = serializers.FloatField(label=_("Latitude"))
+    lng = serializers.FloatField(label=_("Longitude"))
+
+
+class MapSearchRD(serializers.Serializer):
+    x = serializers.FloatField(label=_("X"))
+    y = serializers.FloatField(label=_("Y"))
```

Comment:
It this the RijksDriehoek projection of the same location?
Do we use it?

From the mock in  open-formulieren/open-forms-sdk#486 it doesn't seem we use it (doest leaflet even support it?), yet in the processing of the pdok response, we require them to be present...

I ask because:

Since they are both different representations a point in space, what does it mean when they point to different points? So I'd prefer serving 1 representation and applying a transform on the client.

Reply:
@sergei-maertens asked me to expose it, because Leaflet works with RijksDriehoek although doesn't support the usage of it at the moment.

Reply:
ideally we'd use the RD coordinates only at some point because there's always precision loss with projections. I don't want to get into the complexities of Leaflet on that aspect right now because it's very confusing, but it shouldn't hurt to expose both WSG:84 and RD coordinates _if we have them_.